core/domain/suggestion_services.py

```diff
@@ -588,13 +588,25 @@ def get_all_suggestions_that_can_be_reviewed_by_user(user_id):
     ])
 
 
-def get_reviewable_suggestions(user_id, suggestion_type):
+def get_reviewable_suggestions(user_id, suggestion_type,
+                               opportunity_summary_exp_ids, topic_name):
     """Returns a list of suggestions of given suggestion_type which the user
-    can review.
+    can review. If suggestion_type is translate_content, we only fetch
+    those suggestions which match with the passed opportunity ids.
 
     Args:
         user_id: str. The ID of the user.
         suggestion_type: str. The type of the suggestion.
+        opportunity_summary_exp_ids: list(str).
```

Comment:
See notes above; this design of the function's API is too complicated.

Reply:
This is done.